migrations/__tests__/00004_notifications.spec.ts

```diff
@@ -0,0 +1,615 @@
+          },
+          {
+            id: expect.any(Number),
+            name: 'MESSAGE_CREATED',
+          },
+          {
+            id: expect.any(Number),
+            name: 'MODULE_TRANSACTION',
+          },
+          {
+            id: expect.any(Number),
+            name: 'NEW_CONFIRMATION',
+          },
+          {
+            id: expect.any(Number),
+            name: 'MESSAGE_CONFIRMATION',
+          },
+          {
+            id: expect.any(Number),
+            name: 'OUTGOING_ETHER',
+          },
+          {
+            id: expect.any(Number),
+            name: 'OUTGOING_TOKEN',
+          },
+          {
+            id: expect.any(Number),
+            name: 'PENDING_MULTISIG_TRANSACTION',
+          },
+          {
+            id: expect.any(Number),
+            name: 'SAFE_CREATED',
+          },
+        ],
+      },
+      notification_channels: {
+        columns: expect.arrayContaining([
+          { column_name: 'id' },
+          { column_name: 'name' },
+        ]),
+        rows: [
+          {
+            id: expect.any(Number),
+            name: 'PUSH_NOTIFICATIONS',
+          },
+        ],
+      },
+      notification_channel_configurations: {
+        columns: expect.arrayContaining([
+          { column_name: 'updated_at' },
+          { column_name: 'created_at' },
+          { column_name: 'notification_subscription_id' },
+          { column_name: 'notification_channel_id' },
+          { column_name: 'id' },
+          { column_name: 'device_uuid' },
+          { column_name: 'cloud_messaging_token' },
+        ]),
+        rows: [],
+      },
+    });
+  });
+
+  it('should upsert the row timestamps of notification_subscriptions on insertion/update', async () => {
+    const afterInsert = await migrator.test({
+      migration: '00004_notifications',
+      after: async (sql: postgres.Sql) => {
+        await sql.begin(async (transaction) => {
+          // Create account
+          await transaction`INSERT INTO accounts (address)
+                                VALUES ('0x69');`;
+          // Add notification subscription to account
+          await transaction`INSERT INTO notification_subscriptions (account_id, chain_id, safe_address)
+                                VALUES (1, 1, '0x420')`;
+        });
+
+        return {
+          columns:
+            await sql`SELECT COLUMN_NAME FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = 'notification_subscriptions'`,
+          rows: await sql<
+            Array<NotificationSubscriptionsRow>
+          >`SELECT * FROM notification_subscriptions`,
+        };
+      },
+    });
+
+    expect(afterInsert.after).toStrictEqual({
+      columns: expect.arrayContaining([
+        { column_name: 'id' },
+        { column_name: 'account_id' },
+        { column_name: 'chain_id' },
+        { column_name: 'safe_address' },
+        { column_name: 'created_at' },
+        { column_name: 'updated_at' },
+      ]),
```

Comment:
Nit: do you think is necessary to check the `columns` also on the upsert-related tests? (they are already checked in the migration test itself (`runs successfully`))

Reply:
Removed in bbf887220263d3371eb3bbe8021baa16e7b1470c.